charts/linstor-scheduler/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.podDisruptionBudget.enabled -}}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "linstor-scheduler.fullname" . }}
+  labels:
+    {{- include "linstor-scheduler.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "linstor-scheduler.labels" . | nindent 6 }}
+  {{- if .Values.podDisruptionBudget.minAvailable }}
+  minAvailable: {{ .Values.podDisruptionBudget.minAvailable }}
+  {{- end }}
+  {{- if .Values.podDisruptionBudget.maxUnavailable }}
+  maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
+  {{- end }}
```

Comment:
If  both`replicaCount` of deployment is set to `1` and `minAvailable` of podDistruptionBudget is set to `1`, the updating process will most probably stuck forever.

Reply:
Sorry, I was probably wrong, because with `StrategyType: RollingUpdate` it can generate new replica before the deleting the old one

Reply:
Tested it, doesn't seem to be an issue